reproduction-dependencies/Directory.Build.props

```diff
@@ -0,0 +1,23 @@
+<Project>
```

Comment:
Can you add this file to the `reproduction-dependencies` folder in the solution? Otherwise it's not obvious when working inside VS.

Reply:
Fixed!